fdbclient/include/fdbclient/ServerKnobs.h

```diff
@@ -747,6 +747,7 @@ class ServerKnobs : public KnobsImpl<ServerKnobs> {
 	int64_t EMPTY_READ_PENALTY;
 	int DD_SHARD_COMPARE_LIMIT; // when read-aware DD is enabled, at most how many shards are compared together
 	bool READ_SAMPLING_ENABLED;
+	int64_t BYTES_AVERAGE_INTERVAL;
```

Comment:
It looks like most intervals in this file are `int`s or `double`s. Can this interval be an `int` instead?

Reply:
I found most `*interval` knobs using `double`(so as `*duration` knobs). Changed it to `double` as well